src/components/subsetDraft/SubsetMetadata.jsx

```diff
@@ -31,32 +44,34 @@ export const SubsetMetadata = ({subset}) => {
                                   size={{cols: 65, rows: 1}}
                                   prefix={subsetDraft.namePrefix}
             />
-
+            
             <section style={{margin: '5px 0 5px 0'}}>
                 <div style={{float: 'left', marginRight: '20px', padding: '0'}}>
                     <label style={{display: 'block', fontSize: '16px', fontFamily: 'Roboto'}}
                            htmlFor="from_date">{t('Valid from')}: </label>
                     <DatePicker id='from_date' style={{display: 'block'}}
                                 value={subset.draft.validFrom}
-                                onChange={(date) => subset.dispatch({action: 'from', data: date})}
+                                onChange={(date) => warnBeforeChangingDate(date, 'from') }
```

Comment:
Yes this is due to the fact that I went for an infobox solution since SSB does not have a modal solution in their style guides. I`ll build a modal that we can use instead of the solution I went for.

Reply:
Three buttons are a little too much buttons? Right? :)

Reply:
Right 😅 
We go for `Remove` and `Cancel`